go_api/main.go

```diff
@@ -43,6 +43,12 @@ func main() {
 
 	userController := controller.NewUserController(userUsecase)
 
+	adminInfrastructure := infrastructure.NewAdminInfrastructure(client)
+
+	adminUsecase := usecase.NewAdminUsecase(adminInfrastructure)
+
+	adminController := controller.NewAdminController(adminUsecase)
+
```

Comment:
```suggestion
	userInfrastructure := infrastructure.NewUserInfrastructure(client)
	adminInfrastructure := infrastructure.NewAdminInfrastructure(client)
	
	userUsecase := usecase.NewUserUsecase(userInfrastructure)
	adminUsecase := usecase.NewAdminUsecase(adminInfrastructure)

	userController := controller.NewUserController(userUsecase)
	adminController := controller.NewAdminController(adminUsecase)
```
上記のような書き方の方が可読性が向上しそうですね